python/paddle/fluid/tests/unittests/dygraph_to_static/yolov3.py

```diff
@@ -314,19 +314,21 @@ def forward(self,
                         scores, perm=[0, 2, 1]))
             self.downsample //= 2
 
-        if not self.is_train:
-            # get pred
-            yolo_boxes = fluid.layers.concat(self.boxes, axis=1)
-            yolo_scores = fluid.layers.concat(self.scores, axis=2)
-
-            pred = fluid.layers.multiclass_nms(
-                bboxes=yolo_boxes,
-                scores=yolo_scores,
-                score_threshold=cfg.valid_thresh,
-                nms_top_k=cfg.nms_topk,
-                keep_top_k=cfg.nms_posk,
-                nms_threshold=cfg.nms_thresh,
-                background_label=-1)
-            return pred
-        else:
-            return sum(self.losses)
+        # TODO(liym27): Uncomment code after "return" statement can be transformed correctly.
```

Comment:
Why is this return commented? Convert_if shouldn't transform because self.is_train is not tensor.

Reply:
The same as bolow

Reply:
Discussed offline, we will add it back after we support early return.